### DISCUSSION TITLE
Portugol Web não abre documentações de ajuda nem exemplos

### DISCUSSION THREAD

**Discussion by DJFCoder** [dgadelha/Portugol-Webstudio#199] [Dúvidas / Perguntas] [answered]
Olá, boa noite. Não estou conseguindo abrir/ter acesso a documentação de ajuda nem abrir os exemplos

![image](https://github.com/dgadelha/Portugol-Webstudio/assets/147553362/77622c96-788d-4d92-af63-cfba125d15b5)

![image](https://github.com/dgadelha/Portugol-Webstudio/assets/147553362/05ab1a36-9d39-4c48-9936-12e97d21c037)

**Comment by dgadelha** [comment]
Obrigado pelo report!

**Reply by dgadelha** [reply, marked as answer]
Corrigido. Atualização deve subir nos próximos 5 - 10 minutos.
